### ISSUE TITLE
payment question

### ISSUE THREAD

**Issue by jimmyhackers** [nicehash/NiceHashMiner#210]
Im currently mining at a rather low rate. ill be lucky to get 0.01 bitcoin every month.

As Im mining in cryptonight n dagger mainly and 4 others minorly i have multiple currencies adding up to the bitcoin minimum payout total.

is each seperate currency changing into bitcoin and being recorded into my bitcoin total almost instantly as its being mined?

or is it each currency is mined and stored as its own cryptocurrency. then when i have mined enough for a minimum payout i get that current "days/hours" exchange rate for all individual currencies into bitcoin?

thanks in advance for the clarification

**Comment by drkskwlkr** [comment]
> is each seperate currency changing into bitcoin and being recorded into my bitcoin total almost instantly as its being mined?

The short answer is 'yes'. The correct answer is, your payout doesn't have anything to do with what currencies are being mined. You are renting your hash power, for which you are paid in BTC, and whatever altcoin is mined on your hardware doesn't matter to you. What currencies are mined with your hash power is of interest only to the hash power buyer.

Also, you are not bothered by the exchange rate between other currencies and Bitcoin, because NiceHash accepts payments for hash power only in BTC (exchange rate fluctuations are eaten entirely by the buyer). The only concern you should have about exchange rates would be if/when you decide to sell your bitcoin for fiat currency.

NiceHash writes down the BTC reward for every valid share you submit as soon as your work is validated (nearly instantly). The reward is based on the momentary work of the most profitable algorithm (as determined by your local copy of NiceHash miner) performed while fulfilling the highest priced order (as determined by a buyer on the [NiceHash marketplace](https://new.nicehash.com/marketplace)).

These, and many other details of the way NiceHash works, are beautifully explained on the [NiceHash help page](https://new.nicehash.com/help/) which nobody cares to read before posting on GitHub.

**Comment by jimmyhackers** [comment]
thanks for that info. its made it all a lot clearer now

i did look through the help pages but i think i had a case of "looking for apples so hard I couldn't see the oranges"

i might still be looking for apples but im failing to see the "how nicehash makes its profit" page amongst the other help topics

so, to clarify. the 3% fee nicehash takes is the difference between the selling and buying price of hashing power?

**Comment by drkskwlkr** [comment]
> i might still be looking for apples but im failing to see the "how nicehash makes its profit" page amongst the other help topics

There isn't a page called 'How we're making profit' but there is an excellent [NiceHash Fee Schedule](https://new.nicehash.com/help/fees) which does this job.

> so, to clarify. the 3% fee nicehash takes is the difference between the selling and buying price of hashing power?

NiceHash takes a cut on both sides (buyers and sellers). They charge two kinds of fees: (1) service fees and (2) deposit/withdrawal fees. So, much more than 3% in total. More like 6-8% on average.

An additional, and probably inconsequential, source of income for the service comes from one-time new buy order fees and appropriating very small amounts belonging to sellers who no longer use the service (e.g. seller accounts that hold less than 0.0001 BTC and have no recent mining activity).

Basically you shouldn't be wondering why the NiceHash support and social team is so nice and why development and improvements move so fast. This is an admirably profitable service; I guesstimate they bring several hundred thousand dollars per month in gross profits.

**Comment by DillonN** [comment]
Yes NH makes profit off the fees to sellers and buyers, as well as transactions. Thanks for the explanations @drkskwlkr, I'm gonna close this now since it is a NH service question

**Comment by jimmyhackers** [comment]
thanks for the clarification.
i think thery're awesome too.

i wish id found out about all this earlier. my rig would of payed for itself by now :D

keep up the good work guys :)
